apm-agent-plugins/apm-spring-webmvc-plugin/src/main/java/co/elastic/apm/agent/spring/webmvc/ViewRenderInstrumentation.java

```diff
@@ -106,9 +106,14 @@ public static String defineSubtype(String className) {
                 case "org.thymeleaf.spring4.view.ThymeleafView":
                     return "Thymeleaf";
                 default:
-                    break;
+                    if (subTypeCache.containsKey(className)) {
```

Comment:
Usually it's better get and check if `null`, as in:
```
String subType = subTypeCache.get(className);
if (subType == null) {
    subType = className.substring(className.lastIndexOf('.') + 1, className.lastIndexOf("View"));
    if (subTypeCache.size() < 1000) {
        subTypeCache.put(className, subType);
    }
}
return sunType;
```
Then it's one lookup in the cache instead of two every time